infrastructure/README.md

```diff
@@ -104,3 +105,86 @@ Now you can run `kubectl` commands, e.g.
 ```sh
 kubectl get nodes
 ```
+
+## Installing Istio
+
+1. Change to the root directory
+
+   ```sh
+   cd ~
+   ```
+
+2. Specify the [Istio version](https://github.com/istio/istio/releases/)
+
+   ```sh
+   ISTIO_VERSION=1.4.0
+   ```
+
+3. Download and install
+
+   ```sh
+     curl -L https://istio.io/downloadIstio | sh -
+   ```
+
+4. To configure the `istioctl` client tool
+
+   ```sh
+   export PATH="\$PATH:/Users/pataruco/istio-1.4.3/bin"
```

Comment:
might wanna change `/Users/pataruco` for `~`

Reply:
tried it and that doesn't work, it's a literal "~"
`"\$PATH:${HOME}/istio-1.4.3/bin"` should work fine

Reply:
it also has to be `${PATH}` I think - I've just broken my shell

Reply:
sorry for the lack of clarity: the `\` in `\$` creates a literal `$` so the user's path is effectively ruined 👍